requirements.txt

```diff
@@ -22,14 +22,9 @@ pytest
 python-dateutil
 pyvirtualdisplay
 s3fs
+selenium
```

Comment:
I'm going to hold off on adding a version restriction here till we're stable on which version of firefox we support by default. If we restrict the version here, the maintenance cost is to periodically check compatibility and pay tech debt. If we don't, the maintenance cost is to resolve breakage when it happens - I prefer the latter because if and when things break, we can simply restrict the version then, and then we *know* that we have tech debt to pay.